app/js/actions/draftActions.js

```diff
@@ -0,0 +1,13 @@
+import {
+  removeTestFromDraft,
+  removeTestPanelFromDraft,
+  deleteDraftLabOrder,
+} from '../actions/draftLabOrderAction';
+
+const discardTestsInDraft = (test = {}) => (dispatch) => {
+  if (test.draftType === 'single') return dispatch(removeTestFromDraft(test.order));
```

Comment:
what happens here if a string or something that is not an object is mistakenly passed to this function? It would be ignored right?

Reply:
You have to acknowledge this action is called in an on event handler, and the data passed is on the draft list which is either an empty object or an object with properties.

Reply:
Ok